books/models.py

```diff
@@ -387,6 +387,22 @@ def get_subject_meta(self):
         APIField('subject_search_description')
     ]
 
+class k12SubjectBooks(models.Model):
```

Comment:
Please start python classes with a capital letter, so here: `K12SubjectBooks`. Won't call out specifics in the rest of the PR.